package/src/index.ts

```diff
@@ -206,6 +235,50 @@ export class CandleClient {
     });
   }
 
+  private convertTradeAssetRef(
```

Comment:
I don't think we need this function at all? TradeAssetRef is a subtype of (i.e. inherits from) InternalTradeAssetRef, so you can pass a `TradeAssetRef` anywhere that requires an `InternalTradeAssetRef`

Reply:
Type 'import("/Users/gary/Documents/gary-new/candle-react-native/package/src/index").TradeAssetRef' is not assignable to type 'import("/Users/gary/Documents/gary-new/candle-react-native/package/src/specs/RNCandle.nitro").TradeAssetRef'.
  Type '{ assetKind: "transport"; } & TransportAssetRef' has no properties in common with type 'TradeAssetRef'.ts(2322)
RNCandle.nitro.ts(392, 3): The expected type comes from property 'lost' which is declared here on type 'TradeRef'

I get an error. They are different types.